set.mm

```diff
@@ -145359,16 +145363,15 @@ notation is used by Donald Knuth for iterated exponentiation (_Science_
     NVBUSJUMUNVBVAUSUNUPKVAUNBIBUGUHUIUNUQUSUTBUJUKTULT $.
 
   $( Value of a complex number raised to the 0th power.  Note that under our
-     definition, ` 0 ^ 0 = 1 ` , following the convention used by Gleason.
-     Part of Definition 10-4.1 of [Gleason] p. 134.  (Contributed by NM,
-     20-May-2004.)  (Revised by Mario Carneiro, 4-Jun-2014.) $)
+     definition, ` 0 ^ 0 = 1 ` ( ~ 0exp0e1 ) , following the convention used by
```

Comment:
"( ~ 0exp0e1 )" -> "(see ~ 0exp0e1 )"

Reply:
I though about it, and when the expression matches exactly the theorem, I think "see" is not necessary (there are about 800 occurrences of " ( ~ " in set.mm, even if not fall in the same category).